forms/MoneyField.php

```diff
@@ -56,9 +56,7 @@ public function __construct($name, $title = null, $value = "") {
 	 */
 	public function Field($properties = array()) {
 		return
-			"<div class=\"fieldgroup\">" .
-			"<div class=\"fieldgroup-field\">" . $this->fieldCurrency->SmallFieldHolder() . "</div>" .
-			"<div class=\"fieldgroup-field\">" . $this->fieldAmount->SmallFieldHolder() . "</div>" .
+			"<div class=\"form__fieldgroup\">" . $this->fieldCurrency->SmallFieldHolder() . $this->fieldAmount->SmallFieldHolder() .
```

Comment:
I'm thinking about this specifically, can't this be in the MoneyField_holder template?

Also, `$properties` is ignored, which is no fun